hr_attendance_report_theoretical_time/readme/CREDITS.rst

```diff
@@ -0,0 +1,3 @@
+**Images**
+
+* Font Awesome: `Icon <http://fontawesome.io>`_.
```

Comment:
Is this needed? It's already included by Odoo, right?

Reply:
Yes, because the icon is made from one Font Awesome glyph and I give credits for that.